Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/overview/items/PersonCertificateCard.kt

```diff
@@ -0,0 +1,55 @@
+package de.rki.coronawarnapp.covidcertificate.person.ui.overview.items
+
+import android.graphics.Bitmap
+import android.view.ViewGroup
+import androidx.core.graphics.drawable.DrawableCompat
+import de.rki.coronawarnapp.R
+import de.rki.coronawarnapp.covidcertificate.common.certificate.CwaCovidCertificate
+import de.rki.coronawarnapp.covidcertificate.person.ui.overview.PersonOverviewAdapter
+import de.rki.coronawarnapp.covidcertificate.person.ui.overview.PersonOverviewItemColor
+import de.rki.coronawarnapp.databinding.ItemPersonOverviewBinding
+import de.rki.coronawarnapp.util.ContextExtensions.getColorCompat
+import de.rki.coronawarnapp.util.ContextExtensions.getDrawableCompat
+import de.rki.coronawarnapp.util.lists.diffutil.HasPayloadDiffer
+
+class PersonCertificateCard(parent: ViewGroup) :
+    PersonOverviewAdapter.PersonOverviewItemVH<PersonCertificateCard.Item, ItemPersonOverviewBinding>(
+        R.layout.home_card_container_layout,
```

Comment:
Should we add a new layout file for this?

Reply:
why? I thought all cards in home (Main tabs) should be consistent and this btw were used in Certificates cards